assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -18,6 +18,7 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { omit, values } from 'lodash';
```

Comment:
I think I heard there was a decision not to use `lodash`. Elsewhere I'm seeing `objectWithout` utils and such.

/cc @barklund

Reply:
See https://xwp.slack.com/archives/CCCAUAH6F/p1576519153039000 for that discussion.